### PR TITLE
Fixed the lazy tree creation, clean up the code from non-working pref…

### PR DESCRIPTION
**Pull request by bilgili**
…etching

### CODE REVIEW

**Comment by bilgili** [comment]
@tribal-tec FYI

File: livre/Lib/Uploaders/DataUploadProcessor.cpp [hunk 1]
```diff
@@ -200,47 +197,35 @@ void DataUploadProcessor::runLoop_( )
...
-bool DataUploadProcessor::_loadPrioritizedData( const Frustum& frustum,
-                                                const LoadPriority priority )
+void DataUploadProcessor::_loadData()
 {
+    const DashRenderStatus& renderStatus = _dashTree->getRenderStatus();
+
+    const Frustum& frustum = renderStatus.getFrustum();
+    _currentFrameID = renderStatus.getFrameID();
+
+    DFSTraversal traverser;
```
Comment (tribal-tec) [inline livre/Lib/Uploaders/DataUploadProcessor.cpp]: why moved?

File: livre/Lib/Visitor/LODSelectionVisitor.cpp [hunk 2]
```diff
@@ -53,38 +51,15 @@ void LODSelectionVisitor::visit( DashRenderNode& renderNode, VisitState& state )
...
+    const bool isVisible = lodFrustum_.boxInSubFrustum( worldBox, index );
+    renderNode.setVisible( isVisible );
 
-        const ConstCacheObjectPtr texture = renderNode.getTextureObject();
-        if( lodNode.getRefLevel() == 0 &&
-            !texture->isLoaded() )
-            renderNode.setRequested( requestType_, true );
+    const ConstCacheObjectPtr texture = renderNode.getTextureObject();
+    if( lodNode.getRefLevel() == 0 &&
+        !texture->isLoaded() )
```
Comment (tribal-tec) [inline livre/Lib/Visitor/LODSelectionVisitor.cpp]: move to previous line. Or use braces. Can't read on first glance :)

**Comment by tribal-tec** [comment]
+1

File: livre/Lib/Uploaders/DataUploadProcessor.cpp (hunk 1 above)
Comment (bilgili) [inline livre/Lib/Uploaders/DataUploadProcessor.cpp]: _loadPrioritizedData is long gone, long time ago, far far away galaxy there was lots of code for prefetching which is currently useless.

**Comment by bilgili** [comment]
retest this please

File: livre/Lib/Uploaders/DataUploadProcessor.cpp (hunk 1 above)
Comment (tribal-tec) [inline livre/Lib/Uploaders/DataUploadProcessor.cpp]: No, I mean the declaration of the traverser variable. Can be moved (back) to line 216.

Comment (bilgili) [inline livre/Lib/Uploaders/DataUploadProcessor.cpp]: Done

**Comment by bilgili** [comment]
retest this please

**Pull request merged**